lms/static/coffee/src/navigation.js

```diff
@@ -0,0 +1,70 @@
+var edx = edx || {},
```

Comment:
I'm not sure how to remove things from the pipeline, but this is related to that question I had this morning in `dev`. Thanks! Oh, are you reviewing in place of @AlasdairSwan?